src/core/utils/helpers/general.ts

```diff
@@ -405,6 +405,44 @@ export const filterLoansNotOverdue = (loans: LoanType[], warning: number) => {
   });
 };
 
+function getDateFromCpr(cprInput: string) {
+  const cpr = cprInput.replace(/[^\d]/g, "");
+  const dateSegments = cpr.substring(0, 6).match(/.{1,2}/g);
+
+  if (dateSegments) {
+    const [day, month, year] = dateSegments;
+    let prefix = "";
+    if (Number(year) < 21) {
+      prefix = "20";
+    } else {
+      prefix = "19";
+    }
```

Comment:
So this is valid for a little under 100 years. Let's see how long the application stands ;)